index.html

```diff
@@ -4,8 +4,17 @@
   <meta charset="UTF-8">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
   <script src="main.js" defer></script>
+  <link rel="stylesheet" href="style.css">
   <title>Music Search</title>
 </head>
 <body>
+  <form id='search'>
+    <input type='search' id='input-artist' placeholder="Artist/Band Name" class='input-field'></input>
+    <button class='submit-button' type='sumbit'>Search</button>
+    <audio id='audio-player'></audio>
```

Comment:
Since you're including the audio element with each individual song that gets rendered, you can remove it from your html here.